antenna/app.py

```diff
@@ -229,8 +217,10 @@ def on_get(self, req, resp):
         })
 
 
-def get_app():
-    config = ConfigManager([ConfigOSEnv()])
+def get_app(config=None):
+    """Returns AntennaAPI instance"""
+    if config is None:
+        config = ConfigManager([ConfigOSEnv()])
```

Comment:
This is kind of awkward, but I need to be able to generate an app that doesn't pull configuration from the environment so i don't have to worry about that while testing.